test/Parse/raw_string.swift

```diff
@@ -68,6 +68,12 @@ _ = ##"""
       """##
 // CHECK: "a raw string with \"\"\" in it"
 
+_ = #"""#
+// CHECK: "\""
+
+_ = ###"""##"###
+// CHECK: "\"\"##"
+
```

Comment:
Could you add test cases?:
`##""" foo # "# "##` -> `"\"\" foo # \"# "`
`#"""""#` -> `"\"\"\""`